doc/content/microreactors/mrad/mrad_model.md

```diff
@@ -1,37 +1,164 @@
+
+### Steady State Operation
+
+For steady state simulation, the Griffin parent application performs an Eigenvalue calculation with a fixed preset integrated power (345.6 kW) to compute $k_{eff}$ as well as the shape of power distribution. In the meantime, the BISON child application works with multiple Sockeye grandchild sub-applications to perform transient thermal calculation using the given power density distribution until a thermal equilibrium is achieved. Once the fixed point iteration between different levels of MultiApps is converged, the steady state operation condition of the HP-MR at the given power level is obtained.
+
+Optionally, the fixed preset integrated power level can be raised to force the HP-MR to operate at an overpower level. This is useful to demonstrate the capabilities of the NEAMS codes to predict some types of power transients that only occurs in the case of overpowered operation, such as the heat pipe cascade failure event to be demonstrated in this model.
+
+### Load Following Transient
+
+The load following transient is initiated by introducing a drop in the heat removal capability of the secondary coolant loop that transfer the heat from the condensor regions of the heat pipes to either a heat exchanger or directly to the energy coversion component. During steady state operation, a high heat transfer coefficient (HTC) of 10^6^ W/m^2^•K is used on the outer surface of the heat pipe condensor region. To initiate the load following transient event, the HTC value is reduced by 99.99% to 10^2^ W/m^2^•K to mimic a load reduction event. The same transient solving approach is used for BISON and Sockeye applications. However, instead of an Eigenvalue calculation, Griffin here also performs transient calculation. Based on the changes in $k_{eff}$ induced by temperature evolution calculated by BISON/Sockeye, Griffin predicts the corresponding kinetics of neutronics and consequent power distribution.
+
+### Transient Initiated by a Single Heat Pipe Failure
+
+Another type of transient to be simulated in this model is more localized. Instead of reducing the condensor region HTC of all heat pipes, the condensor region HTC of a single heat pipe is nearly eliminated (i.e. reduced from 10^6^ W/m^2^•K to 10^-2^ W/m^2^•K) to mimic a single heat pipe failure event, while the other heat pipes are kept unchanges (note that actually there are six heat pipes that are compromised due to the 1/6 symmetric mesh used here in this model). The failed heat pipe is intentially selected to be in the hottest region of the HP-MR (i.e., in the center region of the center assembly).
```

Comment:
```suggestion
Another type of transient to be simulated in this model is more localized. Instead of reducing the condensor region HTC of all heat pipes, the condensor region HTC of a single heat pipe is nearly eliminated (i.e. reduced from 10^6^ W/m^2^•K to 10^-2^ W/m^2^•K) to mimic a single heat pipe failure event, while the other heat pipes are kept unchanged (note that actually there are six heat pipes that are compromised due to the 1/6 symmetric mesh used here in this model). The failed heat pipe is intentionally selected to be in the hottest region of the HP-MR (i.e., in the center region of the center assembly).
```